docs/testing-flakes.md

```diff
@@ -0,0 +1,60 @@
+# OpenShift on Azure testing flakes triaging
+
+A lot of end-to-end / conformance tests depend on resources that may vary in
+time with different workloads, software stack versions and available
+hardware. For that reason, a reasonable chunk of our test harness isn't deterministic.
+A side effect of that is that new Pull Requests advancing through the testing pipeline
+may fail with reasons that are unrelated to the changes that comprises that pull request
+but rather by some underlying components that the cluster uses to run these tests.
+Those test failures are called flakes.
+
+The following is intended as a guide for identifying those testing flakes and to
+document them as well as possible so that it's possible to come back at a later
+time and help eliminate them or lower their occurrence rate.
+
+## What to do when a PR testing fails
+
+1. Make sure the error isn't related to the change. When confirmed, it's time to investigate
+   the flake further.
+2. After initial inspection of the flake (See Tricks below), either :
+	* CREATE a new github issue to report a flake never seen before. Try to add
+          some basic info :
+		* Title starting with : "Flake:"
+		* Links to PR where encountered and to CI-operator PR diagnostic page
+                  for the failing test
+		* A brief but complete description of how it was encountered with the last
+                  or most relevant error message
+	* **OR** UPDATE with additional findings a previously created issue for that flake:
+		* Variations in errors messages from where it was encountered before
+		* Complete or partial tracing of to the source of the problem
+		* Links to other PR or external issues referencing this flake
+3. Ensure the flake issue is labeled appropriately:
+```
+/kind test-flake
+```
+4. Back to your PR page, try to get passed the testing failure:
+```
+/retest               # to schedule a new complete test run
+/test <failing test>  # to schedule only the specific test that failed
+```
+
+good luck!
```

Comment:
Need the additional points:
* if you can fix the flake, do so.
* if you can't fix the flake, add judicious logging or saving of artifacts to try to make the flake fixable next time around
* fixing flakes is everyone's responsibility.